actor/echo/src/lib.rs

```diff
@@ -2,6 +2,13 @@ use serde_json::json;
 use wasmbus_rpc::actor::prelude::*;
 use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
 
+#[no_mangle]
+fn _start() {
+    println!("hello from prinln");
+    console_log("hello from console_log");
+}
+
```

Comment:
This looks like a little leftover code from testing tinygo / WASI things, I believe we already intercept the `_start` function in the host and it shouldn't need to be here

Reply:
Yep this is superfluous